src/pages/settings/Security/LockAccount/LockAccountPage.tsx

```diff
@@ -82,11 +78,16 @@ function LockAccountPage() {
                 }}
                 onCancel={() => setIsConfirmModalVisible(false)}
                 isVisible={isConfirmModalVisible}
-                prompt={translate('lockAccountPage.warning')}
+                prompt={
+                    <>
+                        <Text style={[styles.mb5]}>{translate('lockAccountPage.areYouSure')}</Text>
+                        <Text style={[styles.mb5]}>{translate('lockAccountPage.ourTeamWill')}</Text>
+                    </>
+                }
                 confirmText={translate('lockAccountPage.lockAccount')}
                 cancelText={translate('common.cancel')}
                 shouldDisableConfirmButtonWhenOffline
-                shouldShowCancelButton
+                shouldShowCancelButton={false}
```

Comment:
Was this for dev or should we keep it?

Reply:
Oh hmm, I removed it cause the modal looks better without it but I guess actually maybe we should show the cancel button here since we only want users to proceed with locking their account if they're sure 🤔

Reply:
I'll change it back to what it was